### PR TITLE
Fix application dependencies

### PR DESCRIPTION
**Pull request by k32**
Add sext as a runtime dependency, otherwise relx will not include it to the release.

### CODE REVIEW

**Comment by uwiger** [comment]
We are waiting for an update where transaction rollback is mapped.

**Comment by uwiger** [comment]
But will of course add the sext dependency

File: rebar.config [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {deps,
  [
   {sext, "1.8.0"},
-  {rocksdb,"1.6.0"}
+  {rocksdb, {git, "git@github.com:k32/erlang-rocksdb.git", {tag, "1.7.0-k32"}}}
```
Comment (uwiger) [inline rebar.config:7]: Please delete this from the PR

File: src/mnesia_rocksdb.app.src [hunk 2]
```diff
@@ -8,5 +8,5 @@
   {registered, []},
   {mod, {mnesia_rocksdb_app, []}},
   {env, []},
-  {applications, [kernel, stdlib, rocksdb]}
+  {applications, [kernel, stdlib, rocksdb, sext]}
```
Comment (uwiger) [inline src/mnesia_rocksdb.app.src:11]: This is fine

**Review by uwiger — CHANGES_REQUESTED** [review]

**Comment by k32** [comment]
Sorry, I pretty much forgot about this PR and started pushing some experimental changes to the master. Maybe close this PR and merge https://github.com/aeternity/mnesia_rocksdb/pull/6/files instead.

**Comment by uwiger** [comment]
BTW, please take a look at PR #18, which I expect we will merge pretty soon. It's quite a significant rewrite.

**Pull request closed without merging**